Project-I-Social-Card/social-card/src/App.js

```diff
@@ -1,22 +1,13 @@
 import React from 'react';
+import HeaderContainer from './components/HeaderComponents/HeaderContainer';
+import CardContainer from './components/CardComponents/CardContainer';
 import './App.css';
 
-const App = () => {
+const App = (props) => {
   return (
```

Comment:
Great job using props passed from a parent component. 
If you want you can as well deconstruct the props object in this fashion `App = ({ data }) => {}`
It's not compulsory but could make the body of your component cleaner.